fastlane/Fastfile

```diff
@@ -109,6 +109,8 @@ platform :ios do
     commit_hash = last_git_commit[:commit_hash]
     puts commit_hash
 
+    is_prerelease = new_version_number.split("-").length > 1
+
```

Comment:
This isn't exactly fancy, but it does the job. 

A fancier alternative would be to use a [gem like sem-version](https://github.com/canton7/sem_version) to actually parse the version number, but it kinda feels like overkill right now. Happy to update if others feel it's important, though.